src/Features/Core/Portable/Completion/Providers/AbstractObjectInitializerCompletionProvider.cs

```diff
@@ -84,6 +84,7 @@ protected override Task<CompletionDescription> GetDescriptionWorkerAsync(Documen
         private static bool IsLegalFieldOrProperty(ISymbol symbol)
         {
             return symbol.IsWriteableFieldOrProperty()
+                || symbol.ContainingType.IsAnonymousType
                 || CanSupportObjectInitializer(symbol);
```

Comment:
Not sure if the new check should be moved to `CanSupportObjectInitializer`?

Reply:
i'm fine with either approach.